client/app/src/Controller/Admin/OrganisationController.php

```diff
@@ -66,6 +70,8 @@ public function viewAction(Request $request, int $id)
         $tab = $request->get('tab', 'users');
         $currentFilters = self::getFiltersFromRequest($request);
 
+        $clientCount = $this->restClient->get('/v2/organisation/'.$id.'/clients', 'array')['count'];
```

Comment:
I think we will be making nearly the same API call twice if we are on the clients tab. Is that the case? Does the request which includes filters (line 75) return the total client count? If it does, we don't need to do the call in 73, as `$result` already contains the data.

If two calls are required, we should only do the line 73 call inside the conditional (line 77), as the data is otherwise not going to be used.